swayfmt/src/items/item_configurable/mod.rs

```diff
@@ -94,24 +91,17 @@ impl Format for ItemConfigurable {
                                 }
                             }
                             // Add `:`, `ty` & `CommaToken`
-                            write!(
-                                formatted_code,
-                                " {} ",
-                                configurable_field.colon_token.ident().as_str(),
-                            )?;
+                            write!(formatted_code, " {} ", ColonToken::AS_STR,)?;
```

Comment:
```suggestion
                            write!(formatted_code, " {} ", ColonToken::AS_STR)?;
```

Reply:
Good catch! I've run a search for `,)` and found a several more occurrences. All are removed now.